italian/homepage.md

```diff
@@ -2,40 +2,33 @@
 
 ### L'obiettivo del nostro lavoro è fornire informazioni rilevanti che risultino in azioni pragmatiche per i singoli individui, le aziende, in governi e le ONG.
 
-_sono un fisico con esperienza al MIT e uno studioso della complessita` che si occupa di pandemie. Ho avvisato del rischio di pandemie globali legate ai viaggi crescenti per 15 anni. Ho raccomandato il controllo di sintomi nelle comunità per fermare l'Ebola in Africa Occidentale nel 2014, ed ha funzionato. Il modo piu' veloce, e anche l'unico, di fermare COVID-19 e' un contenimento nazionale di cinque settimane._
+_Sono un fisico con esperienza al MIT e uno studioso della complessità che si occupa di pandemie. Ho avvisato del rischio di pandemie globali legate alla maggiore connettività globale per 15 anni. Ho raccomandato il controllo di sintomi nelle comunità per fermare l'**Ebola** in Africa Occidentale nel 2014, ed ha funzionato._
+
+Unisciti al nostro team di più di 5000 volontari nella lotta contro COVID-19: (LINK TO SIGNUP PAGE)
 
 |![yaneer](../0_english_source/images/Yaneer.jpg)|
 |Prof. Yaneer Bar-Yam, PresidentNECSI|
 | --- |
 
-### La chiave per fermare la diffusione del virus è la diminuzione della connettività
-
-Ogni volta che il virus non infetta una persona in più, c’è un effetto sostanziale nella riduzione generale delle infezioni. Quanto più informati siamo sui comportamenti da adottare, tanto meglio saremo in grado di proteggere i nostri cari più vulnerabili e la nostra infrastruttura sanitaria.
-
-Senza interventi, si stima che un individuo con il virus potrà infettare in media altre 3-4 persone, determinando un incremento esponenziale del numero dei casi nel tempo. Se riusciremo a ridurre le relazioni sociali in modo tale che il numero di coloro che vengono contagiati da ogni persona infetta scenda sotto l’unità, il numero di contagi diminuirà  esponenzialmente!
-
-Tenetevi informati sulle ultime notizie per prevenire e combattere la diffusione di COVID-19, scritte e pubblicate da esperti nel settore.
-
-### Unisciti a noi nella lotta contro COVID-19
-
-(LINK TO SIGNUP PAGE)
```

Comment:
I couldn't see this anywhere in [the original doc](https://github.com/necsi/source-translation-text/blob/master/0_english_source/homepage.md)